turbinia/workers/plaso.py

```diff
@@ -25,13 +25,61 @@
 from turbinia.evidence import EvidenceState as state
 from turbinia.evidence import PlasoFile
 from turbinia.workers import TurbiniaTask
+from turbinia.lib import file_helpers
 
 
 class PlasoTask(TurbiniaTask):
   """Task to run Plaso (log2timeline)."""
 
   # Plaso requires the Disk to be attached, but doesn't require it be mounted.
-  REQUIRED_STATES = [state.ATTACHED, state.DECOMPRESSED]
+  REQUIRED_STATUS = [state.ATTACHED, state.DECOMPRESSED]
+
+  task_config = {
+      'status_view': 'none',
```

Comment:
Should this be just `None` similar to below?

Reply:
Let's make sure that the None vs. 'none' issue is consistent throughout how it's defined in the recipes, implemented in the TurbiniaTask.task_config variables, and how it's called from the `TurbiniaTask.run()` methods.

Reply:
re this and the next comment, this would be the default value as specified in the Plaso help. "none" seemed more appropriate for the circumstances of a turbinia worker.

--status_view TYPE, --status-view TYPE                                                                            
                        The processing status view mode: "linear", "none" or "window".

Reply:
Yeah, I think 'none' as a string is the correct thing here then.  As mentioned below, let's maybe add a comment about that though since it could be easy to get wrong.